### DISCUSSION TITLE
import alpha.data中的TrainValData和TimeSeriesData时出现Segmentation fault

### DISCUSSION THREAD

**Discussion by markone1** [Congyuwang/AlphaNetV3#26] [General]
您好，

您的代码对我帮助很大，但是当我在linux系统里面跑您的代码时，在import阶段出现了Segmentation fault，导致程序直接退出。

这个问题是什么原因呢？

**Comment by Congyuwang** [comment]
你好。segmentation fault是底层的内存错误，也就是说是某个依赖的包底层C/C++代码的错误。你可以具体看一下是import到哪一步的时候出的错。

**Reply by Congyuwang** [reply]
也就是说一行一行地import，看是哪一个导致的错误

**Reply by markone1** [reply]
我是import到alpha.data TrainValData TimeSeriesData时出的错

**Reply by Congyuwang** [reply]
那么seg fault也不可能是python代码的错误。除了可能是依赖包的错误，也有可能是你使用的编辑器，运行环境之类的错误。

**Reply by markone1** [reply]
好的，十分感谢！我去检查一下。
